src/components/views/ViewColumnDialog/index.tsx

```diff
@@ -22,7 +22,7 @@ interface ViewColumnDialogProps {
 }
 
 const ViewColumnDialog : FunctionComponent<ViewColumnDialogProps> = ({ selectedColumn, onCancel, onSave }) => {
-    const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+    const isMobile = useMediaQuery(lightTheme.breakpoints.down('sm'));
```

Comment:
Rather than import the `lightTheme` inside each component, the "proper" way to access the currently in use theme is to use the `useTheme` hook, which I think we just haven't used so far.

https://mui.com/styles/advanced/#accessing-the-theme-in-a-component

Reply:
You're right, I will make that change!